graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java

```diff
@@ -0,0 +1,39 @@
+package com.expediagroup.streamplatform.streamregistry.graphql.model.inputs;
+
+import static com.expediagroup.streamplatform.streamregistry.graphql.model.inputs.NameNormaliser.normalise;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class NameNormaliserTest {
+  @Test
+  public void validName() {
```

Comment:
Seems like this filter would accept abc___________def (multiple underscores), which I suspect is not allowable. 
Would be nice to see a test which clarifies and validates this outcome.

Reply:
Good spot - updated regex to restrict consecutive underscores.